### PR TITLE
Add project: use an instance variable to avoid 500

### PR DESCRIPTION
**Pull request by humitos**
Closes https://github.com/readthedocs/ext-theme/issues/477

### CODE REVIEW

**Review by stsewd — COMMENTED** [review]
Looks like you can use a `condition_dict` class attribute to define when to skip a step https://django-formtools.readthedocs.io/en/latest/wizard.html#formtools.wizard.views.WizardView.condition_dict.

**Comment by humitos** [comment]
I implemented it using `condition_dict`. It's not perfect because that method is called multiple times and that's kind of an issue for us due that we perform external API calls.

File: readthedocs/projects/urls/private.py [hunk 1]
```diff
@@ -70,7 +70,10 @@
     ),
     path(
         "import/manual/",
-        ImportWizardView.as_view(),
+        ImportWizardView.as_view(
+            ImportWizardView.form_list,
+            condition_dict={"config": ImportWizardView.show_config_step},
```
Comment (stsewd) [inline readthedocs/projects/urls/private.py:75]: You should be able to declare this as a class attribute instead of passing it to the view.

**Comment by stsewd** [comment]
> It's not perfect because that method is called multiple times and that's kind of an issue for us due that we perform external API calls.

Maybe it was fixed in the latest releases? Or maybe this is an open bug https://github.com/jazzband/django-formtools/issues/134

File: readthedocs/projects/urls/private.py (hunk 1 above)
Comment (humitos) [reply readthedocs/projects/urls/private.py:75]: I used this pattern because it's the one mentioned in the documentation.

**Comment by humitos** [comment]
We can improve/fix that bug later. I want to merge this because a lot of users are hitting this issue when importing projects.

Let's fix the immediate problem first and then we can polish it.

**Comment by humitos** [comment]
Based on this comment https://github.com/jazzband/django-formtools/issues/134#issuecomment-1539873122, the problem is not solved in `master` since they are still re-calculating `all` property by calling `get_form_list()`.

File: readthedocs/projects/urls/private.py (hunk 1 above)
Comment (stsewd) [reply readthedocs/projects/urls/private.py:75]: Yeah, but it's confusing having this logic outside the class.

Comment (humitos) [reply readthedocs/projects/urls/private.py:75]: I had to move the method outside the class.

**Comment by humitos** [comment]
Failing test looks unrelated, `FAILED readthedocs/rtd_tests/tests/test_domains.py::FormTests::test_invalid_domains - ValueError: 'special' does not appear to be an IPv4 or IPv6 address`

**Review by stsewd — APPROVED** [review]
I still think we should try to update this dependency as well.

**Comment by humitos** [comment]
Yes. I'm doing that in another branch already 👍🏼

**Pull request merged**